styles/globals.css

```diff
@@ -1,43 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap');
```

Comment:
We should need this, because we can use Next/font
using 

```css
@import
```

in css blocks the browser while the font is being fetched (bad for performance)

if you need to fetch a font seperately from Next/font, use `<link>` tag not `@import`

Here is the docs for next/font

https://nextjs.org/docs/pages/building-your-application/optimizing/fonts